pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu

```diff
@@ -93,17 +94,30 @@ RUN cd /tmp \
   && make -j64 src.build BUILDDIR=/usr/local \
   && rm -rf /tmp/nccl
 
-# Install EFA along with AWS OPEN_MPI
+# Install EFA along without AWS OPEN_MPI
 RUN mkdir /tmp/efa \
   && cd /tmp/efa \
   && curl -O https://s3-us-west-2.amazonaws.com/aws-efa-installer/aws-efa-installer-${EFA_VERSION}.tar.gz \
   && tar -xf aws-efa-installer-${EFA_VERSION}.tar.gz \
   && cd aws-efa-installer \
   && ./efa_installer.sh -y --skip-kmod -g \
+  && rm -rf $OPEN_MPI_PATH \
   && rm -rf /tmp/efa \
   && rm -rf /tmp/aws-efa-installer-${EFA_VERSION}.tar.gz
 
-RUN echo "pml = ob1" >> $OPEN_MPI_PATH/etc/openmpi-mca-params.conf
+# Install OpenMPI without libfabric support
+RUN mkdir /tmp/openmpi && \
```

Comment:
nit: Please change the formatting of this `RUN` statement block to be similar to the blocks in line 124-144, and 146-148.